atd-etl/app/process_cris_cr3.py

```diff
@@ -59,11 +62,15 @@ def wait(int):
 
 with concurrent.futures.ThreadPoolExecutor(max_workers=1) as executor:
```

Comment:
cool to see concurrency here—although y'all are keeping it capped at 1 worker?

Reply:
That concurrency feature has been disabled as long as I can remember - I think the original author coded it up but it did not perform well in practice. As it stands, with just one worker, it is generally able to finish its daily run in 5-10 minutes, so I think we will be in the green in terms of execution duration for the foreseeable future.

Reply:
Thanks for that context, Frank! 🙏  It was fuzzy to me and I decided on the "code around it" approach, but this is making me wonder if should trim it out since it did take a little bit of thinking to see exactly how the process helpers were being called - especially since it isn't being used. What do y'all think?

Reply:
I think if we arent using it, we should remove it/trim it. Especially if it just complicated understanding what is happening

Reply:
Trimming it gets my vote, but I do think also that it works solidly as a rock as is too, so I'm totally fine either way. 👍